Documentation/flex-to-csi-migration.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Migration
+weight: 11900
+indent: true
+---
+
+# Flex to CSI Migration
+
+In Rook v1.8, the Flex driver has been deprecated. Before updating to v1.8, any Flex volumes created in previous versions of Rook will need to be converted to Ceph-CSI volumes.
+
+The tool [persistent-volume-migrator](https://github.com/ceph/persistent-volume-migrator) will help automate migration of Flex rbd volumes to Ceph-CSI volumes.
+
+## Migration Preparation
+
+1. Rook v1.7.x is required. If you have a previous version of Rook running, follow the [upgrade guide](https://rook.io/docs/rook/v1.7/ceph-upgrade.html) to upgrade from previous releases until on v1.7.x.
+2. Enable the CSI driver if not already enabled. See the [operator settings](https://github.com/rook/rook/blob/release-1.7/cluster/examples/kubernetes/ceph/operator.yaml#L29-L32) such as `ROOK_CSI_ENABLE_RBD`.
+3. Create the CSI storage class to which you want to migrate
+4. Existing volumes should be dynamically provisioned.
```

Comment:
The approach with the flex driver and rbd was to always dynamically provision IIRC. Do we need this line?
```suggestion
4. Volumes must have been dynamically provisioned with the flex driver.
```

Reply:
right, removed